### ISSUE TITLE
[Feature] Add support for more than one folder (--> Shared)

### ISSUE THREAD

**Issue by jacotec** [pulsejet/memories#178]
I'm currently fighting with a limitation: I've set the base folder to "/Pictures" which contains all my iPhone fotos and others (like Lightroom exports) in subfolders. That works great.

Unfortunately I can't include shared folders from others into Memories which are living unter "/Shared/...". Using the root path as a starting point for Memories is not an options, as in many other folders it would find tons of graphics etc. which do not belong into Memories.

I'd love an option to include more folders that a single one ... could be a kind of "Expert configuration", but should be available in the settings GUI, so "nomal" users can configure this as well.

**Comment by pulsejet** [comment]
> I just noticed memories actually goes through all my external storage mounts within my home dir on memories:index, they don't show up in the UI but it does scan them.

That's correct. The folders view will be basically equivalent to the files app, but the timeline path is limited to a single storage. So any nested files with different storages will be ignored.

**Comment by hobronc** [comment]
My use case would also really benefit from the multiple folder for the timeline. I use nextcloud to sync the photos from 2 android device into different folders. If one day you decide to add this option it could be helpfull for some of us.
